tests-e2e/ocw-ci-test-course/nav-links.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { test, expect } from "@playwright/test"
+import { CoursePage } from "../util"
+
+test("Navbar links working properly", async ({ page }) => {
```

Comment:
What about an approach like this? Apart from the hard-coded link names, the rest should be more reliable.

```ts
test("Navbar links working properly", async ({ page }) => {
  const course = new CoursePage(page, "course")
  await course.goto("")

  const linkNames = ["Section 2 Menu Title", "Section 1 Menu Title"]

  for (const linkName of linkNames) {
    const link = page.getByRole('link', {name: linkName})
    const url = await link.getAttribute("href")
    
    await expect(url).toBeTruthy()

    const responsePromise = page.waitForResponse((response) => (
      response.url().endsWith(url!)
    ))
    await link.click()
    const response = await responsePromise

    await expect(response?.status()).toBeLessThan(400)
    await expect(page.url()).toContain(url)
  }
})
```

Reply:
Nice approach